src/main/java/io/jenkins/plugins/jfrog/JfStep.java

```diff
@@ -46,8 +49,19 @@
 @SuppressWarnings("unused")
 public class JfStep extends Builder implements SimpleBuildStep {
     private final ObjectMapper mapper = createMapper();
+    private static final Version MIN_CLI_VERSION_PASSWORD_STDIN = new Version("2.31.3");
     static final String STEP_NAME = "jf";
+
     protected String[] args;
+    //  The current JFrog CLI version in the agent
+    protected Version currentCliVersion;
```

Comment:
To be more clear, let's replace `currentCliVersion` and `isPluginLauncher` with 1 class variable `usePasswordFromStdin`

Reply:
You're right.
In order to make it more clear i've added a new field `usePasswordFromStdin` but kept the CLI version, as it may be handy in the future if we would like to enforce more features by CLI version.